test/assertionsSpec.js

```diff
@@ -1,4 +1,5 @@
 var assert = require('assert')
+var expect = require('chai').expect
```

Comment:
hey I am trying to remove usage of chai and switch to must.js mostly for the reasons here https://github.com/moll/js-must#using-shouldjs-or-chaijs-switch-for-safety